### DISCUSSION TITLE
Which service do you prefer?

### DISCUSSION THREAD

**Discussion by boomzero** [langningchen/XMOJ-Script#72] [Polls]
| 项目       | 小明的OJ 论坛             | XMOJ增强脚本                                 |
| :--------- | :----------------------------------- | :------------------------------------------- |
| 界面       | :+1: 界面很好看                      | :x: 界面不是很好看                           |
| `Markdown` | :+1: 支持                            | :+1: 支持                                    |
| `KaTex` | :+1: 支持                            | :+1: 支持                                    |
| 论坛功能   | :+1: 支持投票、邮件提醒、博客、头像等衍生功能              | :x: 仅支持论坛                               |
| 总体功能   | :x: 仅支持论坛与Blog                 | :+1: 支持ACM排名等多种实用功能               |
| 浏览       | :x: 访问后看到的不是XMOJ域名 | :+1: 访问后看到的域名也是XMOJ                |
| 版本       | :+1: 已经进行了稳定的发布            | :+1: 已经进行了稳定的发布                    |
| 管理       | :+1: 管理内容丰富                    | :x: 无法进行细致的管理                       |
| 隐私       | :x: “小明”可以看到你发的内容         | :+1:  你的浏览记录完全被保护不会被“小明”看到 |
| 稳定性     | :x: 服务器相对稳定 (请勿使用共享办公室WiFi访问)             | :+1: 服务器稳定                     |
| 安全性     | :+1: 使用Cloudflare保护              | :+1: 使用Cloudflare保护                             |
| 服务器开源 | :+1: 在Github上开源                  | :+1: 在Github上开源                          |
| 开发过程   | :x: 套的模板，加入了权限、昵称、CDN等 | :+1: 从零开始手写开发                        |
| 服务器地点 | :+1: 中国贵州贵阳市+Cloudflare泛播                             | :+1: Cloudflare泛播                           |
| 域名       | :x: 使用共享域名（单独域名申请中）                     | :+1: 使用单独域名                             |
| 安装方式   | :+1: 无需安装，直接使用              | :x: 需要安装用户脚本管理器                   
| 账号       | :x: 需要注册账号或者使用第三方登录   | :+1: 与XMOJ账号通用，自动登录                |
| 言论安全性 | :+1: 评论可以被自动过滤              | :x: 评论只能管理员手动删除                   |
| 言论自由度 | :x: 攻击性发言可能会被自动屏蔽             | :+1: 管理员一般不会删除评论                  |
| 网站安全性 | :+1: 使用SSL证书加密                 | :+1: 使用SSL证书加密                         |
| ChatGPT | :x: ChatGPT认为不好 | :+1: ChatGPT认为很好|

**Comment by boomzero** [comment]
@langningchen  @PythonSmall-Q ~快投票~

**Reply by langningchen** [reply]
...

**Reply by PythonSmall-Q** [reply]
我还是支持一下自己的服务吧qaq

**Comment by langningchen** [comment]
> 我还是支持一下自己的服务吧qaq

@PythonSmall-Q 确实应该这样
